Either/Monad

```diff
@@ -3,7 +3,7 @@
 in  let Either = ./Type
 
 in    λ(a : Type)
-    →     ./Applicative a
+    →     ./Applicative/Sequential a
```

Comment:
We should have `Monad/fromPureBind` (or something) that provides default values for `map` and `ap`, then we can extract the `Applicative/Sequential` as mentioned above.